docs/README.md

```diff
@@ -13,6 +13,20 @@ If you don't have `pnpm` installed, you will need to do so first.
 pnpm i
 ```
 
+## Optional Packages
+
+If you would like to include consent management functionality in your local development environment,
```

Comment:
Nit: Since the heading is more generally about optional packages, I would add a sub-heading for this dependency specifically.